cloud/aws/actuators/machine/actuator_test.go

```diff
@@ -249,16 +249,95 @@ func TestUpdate(t *testing.T) {
 	me := mock_ec2iface.NewMockEC2API(mockCtrl)
 	defer mockCtrl.Finish()
 
-	mg.mi.EXPECT().
-		UpdateStatus(&clusterv1.Machine{
-			Status: clusterv1.MachineStatus{
-				ProviderStatus: &runtime.RawExtension{
-					Raw: []byte(`{"kind":"AWSMachineProviderStatus","apiVersion":"awsproviderconfig/v1alpha1"}
-`),
+	// Expects here are annotated to show how they're actually being called.
+	// Called when creating an instance to update.
+	// Called by Create via:
+	// CreateOrGetMachine -> CreateInstance -> RunInstances
+	me.EXPECT().
+		RunInstances(&ec2.RunInstancesInput{
+			ImageId:      aws.String("aws-instance-id"),
+			InstanceType: aws.String(""),
+			MaxCount:     aws.Int64(1),
+			MinCount:     aws.Int64(1),
+			SubnetId:     aws.String(""),
+		}).
+		Return(&ec2.Reservation{
+			Instances: []*ec2.Instance{
+				&ec2.Instance{
+					State: &ec2.InstanceState{
+						Name: aws.String(ec2.InstanceStateNameRunning),
+					},
+					InstanceId:   aws.String("2345"),
+					InstanceType: aws.String("m5.large"),
+					SubnetId:     aws.String("subnet-1"),
+					ImageId:      aws.String("ami-1"),
 				},
 			},
-		}).
-		Return(&clusterv1.Machine{}, nil)
+		}, nil)
+
+	gomock.InOrder(
```

Comment:
I don't see any tags trying to be updated here, please add a test to test the logic written in the actuator